releasenotes/notes/add-labels-kwarg-read-6bb8ef690765ae5f.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    A new kwarg, ``labels``, has been added to the
+    :meth:`retworkx.PyDiGraph.read_edge_list` and
+    :meth:`retworkx.PyGraph.read_edge_list` constructor methods. When this
+    kwarg is set to ``True`` the first 2 elements on a line in an edge list
+    file are treated as string labels instead of integer node indices. For
+    example:
+
+    .. jupyter-execute::
+
+      import tempfile
+
+      import retworkx
+      from retworkx.visualization import mpl_draw
+
+      with tempfile.NamedTemporaryFile("wt") as fd:
+        fd.write('a b 0\n')
+        fd.write('b c 1\n')
```

Is the third argument an edge label or edge weight?

Could this also work?
```suggestion
        fd.write('a b first_edge\n')
        fd.write('b c second_edge\n')
```
It might have been clear as written if I could also see the `mpl_draw` output, but while reviewing, I found `a b 0` confusing

---

Yep, good call this definitely is easier to read as an example.